### DISCUSSION TITLE
Disk monitoring has been fluctuating

### DISCUSSION THREAD

**Discussion by XTUxiongda** [minio/minio#16081] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/38322373/202382647-ad507e97-fd87-4c7a-af4a-4970c26e0fd7.png)
![image](https://user-images.githubusercontent.com/38322373/202382683-52fb084d-81c1-4153-a1ef-adc7d118bd05.png)

Disk monitoring has been fluctuating, changing every time the page is refreshed,  Other indicators are normal
Observe the minio_cluster_disk_offline_total index collected by prometheus, it is indeed changing
No abnormal log, the service is available
version:  2022-08-13
can you give me some advice

**Comment by harshavardhana** [comment, marked as answer]
This issue is related to something that we are investigating in https://github.com/minio/minio/issues/15959

**Reply by XTUxiongda** [reply]
> This issue is related to something that we are investigating in #15959
I have four nodes, and only one node has a problem with the indicator data (minio_cluster_disk_offline_total) collected.
But I used the mc admin info to execute on four nodes, and the data I got was normal
Is there any possible reason for the abnormal indicator data collected by a single node
What are the differences between the mc admin info method and the prometheus collection method, and why the disk online data displayed are inconsistent
